src/KinesisVideoProducer.h

```diff
@@ -41,6 +41,7 @@ namespace com { namespace amazonaws { namespace kinesis { namespace video {
  */
 #define CLIENT_STREAM_CLOSED_CALLBACK_AWAIT_TIME_MILLIS (10 + TIMEOUT_AFTER_STREAM_STOPPED + TIMEOUT_WAIT_FOR_CURL_BUFFER)
 
+#define CONTROL_PLANE_URI_ENV_VAR ((PCHAR) "CONTROL_PLANE_URI")

```

Comment:
Does this belong in the core or in the samples? Would be nice to have the same behavior as in the C Producer - whether elements like these are defined in the main include or in the samples.

Reply:
Using this only in kvssink as of now. In producer C, atleast in the samples, we are using callbacks that allow passing this. Ok either ways.